packages/yarnpkg-cli/sources/tools/WelcomeCommand.ts

```diff
@@ -20,14 +21,15 @@ use the following URLs to get some insight:
   - Our Discord server:
     ${formatUtils.pretty(configuration, `https://discord.gg/yarnpkg`, `cyan`)}
 
-We're hoping you will enjoy the experience. For now, a good start is to run
-the two following commands:
+We hope you will enjoy your experience with Yarn 2.
+
+A good start is to run the two following commands:
 
   ${formatUtils.pretty(configuration, `find . -name node_modules -prune -exec rm -r {} \\;`, `magenta`)}
```

Comment:
We should remove this line as the `node_modules` linker is a thing now and the PnP linker will clean them up anyways

Reply:
Do you think that replacing it by `yarn plugin list` and /or `yarn info` would be of value?  Or even the simple yarn -h?